### PR TITLE
sc - adding entity and repository

### LINKED ISSUE #15: Create database table for MenuItemReview
Create database table for MenuItemReview

# Acceptance Criteria:

- [x] There is an `@Entity` class called MenuItemReview.java
- [x] There is a `@Repository` class called MenuItemReviewRepository.java
- [ ] When you start up the repo on localhost, you can see the table
      using the H2 console (see the file `docs/h2-database.md` for 
      instructions.)
- [x] When you deploy the app to Heroku, you can see the table
      using the psql command (See `Accessing Database Console` in the
      main `README.md` file for instructions.)

### PR DESCRIPTION
**Pull request by sophiacheng01**
In this PR we add a database to the Spring backend capable of storing menu item reviews. This PR adds the required Entity and Repository classes.

Closes #15

### CODE REVIEW

File: src/main/java/edu/ucsb/cs156/example/entities/MenuItemReview.java [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+@Entity(name = "menuitemreview")
+public class MenuItemReview {
+  @Id
+  @GeneratedValue(strategy = GenerationType.IDENTITY)
+  private long id;
+
+  private long itemId;
+  private String reviewerEmail;
+  private int stars;
+  private String comments;
```
Comment (peterlototsky) [inline src/main/java/edu/ucsb/cs156/example/entities/MenuItemReview.java:27]: Maybe the order of Reviews and comments should be switched. In the documentation it seems that the dateReviewed column comes before comments.

**Review by peterlototsky — APPROVED** [review]
LGTM, just one slight change which might not matter.

**Review by bhuang01 — APPROVED** [review]
LGTM

**Review by coding-me-softly — CHANGES_REQUESTED** [review]
~~LGTM, but I think that the MenuItemReview class is missing the `LocalDateTime dateReviewed` column.  I think it requires an import, can check Bryan's entity class for how he did it.~~
Whoops you already had it, I just didn't see it :smiley:

**Review by coding-me-softly — APPROVED** [review]

**Review by dengkevin — APPROVED** [review]
LGTM

**Review by myangichael — APPROVED** [review]
LGTM

**Pull request merged**